packages/lambda-functions/template.yaml

```diff
@@ -12,27 +12,20 @@ Resources:
   HelloWorldFunction:
     Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
     Properties:
-      CodeUri: functions/hello-world
-      Handler: app.lambdaHandler
+      CodeUri: ../../dist/packages/lambda-functions
```

Comment:
We can serve and deploy this function from the dist folder now that it has been built by webpack. The contents of this folder are what will be deployed to lambda